WordPress/src/main/res/layout/new_account_user_fragment_screen.xml

```diff
@@ -87,6 +87,7 @@
                     android:layout_height="wrap_content"
                     style="@style/WordPress.NUXEditText"
                     android:inputType="textUri"
+                    android:digits="0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
You are right, I missed that. I added the check to the site URL on the new user screen and also the new blog page.